docs/source/federation-guide/federated-schema-design.mdx

```diff
@@ -0,0 +1,545 @@
+---
+title: Federated schema design best practices
+sidebar_title: Federated schema design
+---
+
+GraphQL is a relatively new technology, but from its rapid and widespread adoption has emerged a host of common schema design best practices—both from the large, notable companies that use it at scale every day, as well as the broader developer community. The majority of best practices that apply to non-federated GraphQL schema design also apply when designing service schemas within a federated data graph. However, federated schema design rewards some additional best practices when extracting portions of a data graph into subgraphs and determining what extension points to expose between service boundaries.
+
+Throughout this section, we'll explore some proven best practices for GraphQL schema design with a specific lens on how these practices relate to federated data graphs, as well as any special considerations and trade-offs to keep in mind when designing and evolving schemas across a distributed GraphQL architecture.
+
+## Best Practice #1: Start Thinking in Entities
```

Comment:
Should we have a precursor to any of this about teams implementing Federation without entities at first? Like Expedia's transition, they don't have any entities defined and it would be nice to highlight that is possible somewhere and it could even just be a note call out after this

Reply:
Added a note about this in the section.